packages/icons/vite.config.ts

```diff
@@ -0,0 +1,33 @@
+import { defineConfig } from 'vite'
+import path from 'path'
+import react from '@vitejs/plugin-react'
+import dts from 'vite-plugin-dts'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    react(),
+    dts({
+      insertTypesEntry: true,
+      outputDir: 'dist/types',
+    }),
+  ],
+  build: {
+    lib: {
+      // entry: entryObject,
+      entry: path.resolve(__dirname, 'lib/components/index.ts'),
+      name: 'RepublikIcons',
+      formats: ['es'],
+    },
+    rollupOptions: {
+      external: ['react', 'react-dom', 'react/jsx-runtime'],
+      output: {
+        globals: {
+          react: 'React',
```

Comment:
I think we can remove globals if we only build ES?

Reply:
I'd leave it in, as it doesn't hurt us if it's defined.
On the other way around, if we forgot about it, it wouldn't be great.
Therefor I'd keep it arround.